python/repositories.bzl

```diff
@@ -179,6 +179,16 @@ def _python_repository_impl(rctx):
             "share/**",
         ]
 
+    python_wrapper = "python_wrapper.sh"
+    python_wrapper_content = """\
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+external/{repo_name}/{python_bin} -B -s -I "$@"
```

Comment:
I think the "external/" part here might depend on if a flag is set?
https://bazel.build/reference/command-line-reference#flag--legacy_external_runfiles

I think you can drop the "external/" part, but not 100% sure.

Actually, what's the CWD when this is invoked? The relative path here means it would have to be the runfiles root. But what is doing that chdir()? (Sorry if I missed it)

Reply:
Thanks for pointing this out. If `--nolegacy_external_runfiles`, then this path doesn't work. The CWD in my example is

```bash
/home/mvukov/.cache/bazel/_bazel_mvukov/bc9f8eb0563a76ddb0353ce7869ddf8d/execroot/hermetic_python/bazel-out/k8-fastbuild/bin/demo.runfiles/hermetic_python
```
where hermetic_python is the repo name. When `--legacy_external_runfiles` is used, then `external/` has to be used...

> But what is doing that chdir()? (Sorry if I missed it)
I'm not doing any chdir in this PR :) Can you be a bit more specific?